### PR TITLE
Fix tools namespace in merged manifest

### PR DESCRIPTION
**Pull request by marcinkwiatkowski**

### CODE REVIEW

File: buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidTarget.groovy [hunk 1]
```diff
@@ -252,9 +252,11 @@ abstract class AndroidTarget extends JavaLibTarget {
         def builder = new StreamingMarkupBuilder()
         builder.setUseDoubleQuotes(true)
         return (builder.bind {
+            mkp.declareNamespace(tools:'http://schemas.android.com/tools')
```
Comment (kageiit) [inline buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidTarget.groovy]: This should be declared in the input manifest

**Comment by kageiit** [comment]
Seems like CI failed

File: buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidTarget.groovy [hunk 2]
```diff
@@ -254,9 +251,8 @@ abstract class AndroidTarget extends JavaLibTarget {
         return (builder.bind {
             mkp.yield manifestXml
         } as String)
-                .replaceAll('\\{http://schemas.android.com/apk/res/android\\}', 'android:')
-                .replaceAll('xmlns:android="http://schemas.android.com/apk/res/android"', '')
-                .replaceFirst('<manifest ', '<manifest xmlns:android="http://schemas.android.com/apk/res/android" ')
+            .replaceAll('xmlns:android="http://schemas.android.com/apk/res/android"', '')
```
Comment (kageiit) [inline buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidTarget.groovy]: why do we still need these?

Comment (marcinkwiatkowski) [inline buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidTarget.groovy]: This is to handle manifests that don't declare android namespace. The build was previously falling when merging https://github.com/uber/okbuck/blob/master/libraries/emptyResourceLibrary/src/main/AndroidManifest.xml

**Pull request merged**